geode-core/build.gradle

```diff
@@ -15,10 +15,13 @@
  * limitations under the License.
  */
 
-
 apply plugin: 'antlr'
 apply plugin: 'me.champeau.gradle.jmh'
 
+repositories {
```

Comment:
Why was this needed? `mavenCentral` is already a repository for `allprojects`, added in the project root `build.gradle`

Reply:
Sorry -- this might have been added when I temporarily brought in lombok, I realize it's unnecessary. Will remove.